terraform/mod/lxd/main.tf

```diff
@@ -38,7 +38,7 @@ resource "lxd_container" "c8s" {
 
   config = {
     "boot.autostart"       = true
-    "raw.lxc"              = "lxc.mount.auto = proc:rw cgroup:rw sys:rw\nlxc.mount.entry = /lib/modules lib/modules none bind,ro 0 0\nlxc.mount.entry = /boot boot none bind.ro 0 0\nlxc.apparmor.profile = unconfined\nlxc.cgroup.devices.allow = a\nlxc.cap.drop="
+    "raw.lxc"              = "lxc.mount.auto = proc:rw cgroup:rw sys:rw\nlxc.apparmor.profile = unconfined\nlxc.cgroup.devices.allow = a\nlxc.cap.drop="
```

Comment:
Without these mounts, I had to add `SystemVerification` to kubeadm's `--ignore-preflight-errors` in `mod/k8s/master.sh`.

Reply:
that's interesting as I was able to run k8s without any changes even without these filesystems (and @gila too). Normally I would suspect NixOs vs Ubuntu difference but in this case it happens in the container that is the same for all of us. So I can't explain why it fails for you. I have seen error messages about missing kernel modules etc. but they are all accompanied by a follow-up text "please ignore if you run k8s in a container". Are you sure that your kubelet failed because of the missing directories? Perhaps the root cause was a different problem.

Reply:
On my workspace I removed the /boot, which allowed it to proceed to the next step, at which point it failed due to missing `configs` kernel module.  I then added `SystemVerification` to the ignore list.  I would like to see some definitive solution to these difficulties, lest we all end up with a variety of local patches.

Reply:
I started with `SystemVerification` but wasn't sure what other checks it disabled, hence the mount entries. It's looking like it's easier to just add that and move on.

Reply:
Without the mount entries, this is error I get:
`module.k8s.null_resource.k8s[0] (remote-exec): 	[ERROR SystemVerification]: failed to parse kernel config: unable to load kernel module: "configs", output: "modprobe: ERROR: ../libkmod/libkmod.c:586 kmod_search_moddep() could not open moddep file '/lib/modules/5.4.0-0.bpo.4-amd64/modules.dep.bin'\nmodprobe: FATAL: Module configs not found in directory /lib/modules/5.4.0-0.bpo.4-amd64\n", err: exit status 1`

Reply:
The SystemVerification check starts here: https://github.com/kubernetes/kubernetes/blob/release-1.18/cmd/kubeadm/app/preflight/checks.go#L519 and includes kernel config validation: https://github.com/kubernetes/system-validators/blob/v1.1.2/validators/kernel_validator.go#L179

Reply:
Yeah, I would agree, just remove the SystemVerification, otherwise, we have to keep this file generic for all possible permutations and that's really not what we want i would say.

Reply:
I have added SystemVerification flag as suggested by Jonathan. Thanks!